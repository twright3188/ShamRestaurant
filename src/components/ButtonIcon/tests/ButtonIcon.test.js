import React from 'react';
import renderer from 'react-test-renderer';
import { ButtonIcon } from '../index';
import Colors from '../../../theme';

it('icon renders correctly', () => {
    const tree = renderer.create(<ButtonIcon name='ios-person' size={30} color={Colors.activeIcon} />).toJSON();
    expect(tree).toMatchSnapshot();
});