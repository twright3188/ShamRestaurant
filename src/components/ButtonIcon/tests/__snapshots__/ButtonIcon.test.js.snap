// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon renders correctly 1`] = `
<View
  accessible={true}
  clickable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "#399AE5",
          "fontSize": 30,
        },
        undefined,
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï‘¾
  </Text>
</View>
`;
